sdk/purview/azure-analytics-purview-workflow/CHANGELOG.md

```diff
@@ -1,16 +1,20 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2023-10-16)
 
 ### Features Added
 
+- Add `/workflows/{id}/validate` API.
+- Add admin query options to workflow / workflow run APIs.
+
 ### Breaking Changes
 
-### Bugs Fixed
+- Split `PurviewWorkflowClient` into multiple clients by model.
```

Comment:
Would make it easier for people to follow if you specified which clients were broken out from this original one by model.

Reply:
Fixed, thanks.